src/gossip_agent/waku_handling.rs

```diff
@@ -243,28 +243,21 @@ pub fn setup_node_handle(
                 boot_node_id, boot_node_multiaddress
             );
 
-            // Store boot node id to share
-            let mut file = File::create("./boot_node_addr.conf").unwrap();
-            file.write_all(boot_node_multiaddress.as_bytes()).unwrap();
             boot_node_handle
         }
         _ => {
-            let mut file =
-                File::open("./boot_node_addr.conf").expect("Could not find boot_node_addr.conf");
-            let mut boot_node_addr = String::new();
-            file.read_to_string(&mut boot_node_addr)
-                .expect("Could not parse boot node address from boot_node_addr.conf");
-
-            // run default nodes with peers hosted with pubsub to graphcast topics
             info!(
                 "{} {:?}",
                 "Registering the following pubsub topics: ".cyan(),
                 graphcast_topic
             );
 
             let node_config = node_config(host, port, advertised_addr, node_key, false, true);
-            let nodes =
-                Vec::from([Multiaddr::from_str(&boot_node_addr).expect("Could not parse address")]);
+
+            let nodes = env::var("BOOT_NODE_ID")
```

Comment:
Could we have this passed in from the radio init as an option for `static_nodes`?

Reply:
great suggestion! Just added it as another function argument to `GossipAgent::new`, I think that's what you had in mind right? Also, the arguments there are beginning to be a bit too much, in one of my next PRs I think I'll extract them to a config struct (maybe as part of the optional yaml config Issue)